src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -41,18 +42,54 @@ export function AccessCodeDetails({
     )
   }
 
+  const filterCodeErrors = () => {
+    return (
```

Comment:
Better IMO to write this function as a filter function. Can even move it outside the component to the bottom of the file. Need the sdk update for the new types.

```
const errorFilter = (error: AccessCodeError | DeviceError | ConnectedAccountError) => {...}
```

Then later

```
const alerts = [
    ...accessCode.errors.filter(errorFilter).map((error) => ({
      variant: 'error' as const,
      message: error.message,
    })),
```

Reply:
Sounds good, I like this better 👍